src/ApiService/ApiService/Functions/NodeAddSshKey.cs

```diff
@@ -46,8 +46,8 @@ public class NodeAddSshKey {
 
     }
 
-    [Function("node_add_ssh_key")]
-    public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "node/add_ssh_key")] HttpRequestData req) {
+    [Function("NodeAddSshKey")]
+    public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "node_add_ssh_key")] HttpRequestData req) {
```

Comment:
Keep this one, we already had it explicitly specified

```suggestion
    public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "node/add_ssh_key")] HttpRequestData req) {
```